From: =?UTF-8?q?Carl=20F=C3=BCrstenberg?= <carl@excito.com>
Date: Mon, 18 Apr 2011 11:01:19 +0200
Subject: Remove max Audio Scan version

Due to an apparant bug in squeezecenter, it assumes that version 0.880 >
0.88.
Removing the max version for Audio::Scan to solve this.
---
 modules.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.conf b/modules.conf
index 832e845..c1ebdc0 100644
--- a/modules.conf
+++ b/modules.conf
@@ -5,7 +5,7 @@
 # <module> <min_version> [ <max_version> ]
 
 AnyEvent 5.202
-Audio::Scan 0.88 0.88
+Audio::Scan 0.88
 Cache::Cache 1.04
 CGI::Cookie 1.27
 Class::Data::Inheritable 0.04
-- 
